androminion/src/com/mehtank/androminion/fragments/StartGameFragment.java

```diff
@@ -179,7 +202,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         completeSets.put(Expansion.Guilds, mRandomGuilds = (ToggleButton) mView.findViewById(R.id.toggleButtonGuilds));
         completeSets.put(Expansion.Adventures, mRandomAdventures = (ToggleButton) mView.findViewById(R.id.toggleButtonAdventures));
         completeSets.put(Expansion.Empires, mRandomEmpires = (ToggleButton) mView.findViewById(R.id.toggleButtonEmpires));
-        mRandomPromo = (ToggleButton) mView.findViewById(R.id.toggleButtonPromo);
+        completeSets.put(Expansion.Promo, mRandomPromo = (ToggleButton) mView.findViewById(R.id.toggleButtonPromo));
```

Comment:
This isn't supposed to be part of the complete sets, since the Promo cards don't form a complete set. This makes it so you can't start a Random game with just Promo selected (since there wouldn't be enough cards to create a full Kingdom).

Reply:
Ok i'll change this